src/index.js

```diff
@@ -6,45 +6,73 @@ import '@fortawesome/fontawesome-free/js/brands';
 import createEle from './helpers/createEle.js';
 import createTaskArea from './helpers/createTaskArea.js';
 
-const tasks = [
-  {
-    description: 'task 1',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'task 2',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'task 3',
-    completed: false,
-    index: 2,
-  },
-];
+class ToDo {
+  constructor() {
+    this.tasks = [
+      {
+        description: 'task1',
+        completed: false,
```

Comment:
- [ ] Your _imported modules_ do not contain methods related to `status update`. Kindly ensure that your `index.js` has one new JavaScript file imported as a module, which should contain methods related to the `status updates` (completed: true / false).

Reply:
I added the toggleCheck.js file with the corresponding function.